pkg/kubectl/stop.go

```diff
@@ -355,7 +356,16 @@ func (reaper *StatefulSetReaper) Stop(namespace, name string, timeout time.Durat
 	}
 
 	errList := []error{}
-	for _, pod := range podList.Items {
+	for i := range podList.Items {
```

Comment:
Since this is client code, I think we can't do that until 1.7. If we use it against a 1.5 cluster, StatefulSet ObservedGeneration will never catch up.